test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs

```diff
@@ -1,102 +1,142 @@
-            animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(180.0f)), PositionAxis.X), 0, 500);
-            animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(0.0f)), PositionAxis.X), 500, 1000);
+            Animation animation = new Animation(2 * oneSecond);
+            animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(180.0f)), PositionAxis.X), 0, oneSecond);
+            animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(0.0f)), PositionAxis.X), oneSecond, 2 * oneSecond);
             animation.Looping = true;
             animation.Play();
 
-            log.Fatal(tag, "animation play!");
+            log.Debug(tag, "animation play!");
         }
 
-        void CreateSubWin3()
+        private void CreateSubWinThird()
         {
-            if(subWin3)
+            if (subWinThird)
             {
-                log.Fatal(tag, $"Sub Window3 is already created");
+                log.Debug(tag, $"subWinThird is already created");
                 return;
             }
-            subWin3 = new Window("subWin3", new Rectangle(0, 0, 300, 300), false);
-            subWin3.BackgroundColor = Color.Blue;
+            subWinThird = new Window("subWinThird", new Rectangle(0, 0, 3 * testSize, 3 * testSize), false)
+            {
+                BackgroundColor = Color.Blue
+            };
             View dummy = new View()
             {
-                Size = new Size(100, 100),
-                Position = new Position(50, 50),
+                Size = new Size(testSize, testSize),
+                Position = new Position(testPosition, testPosition),
                 BackgroundColor = Color.Yellow,
             };
-            subWin3.Add(dummy);
-            subWin3.KeyEvent += subWin3_KeyEvent;
+            subWinThird.Add(dummy);
+            subWinThird.KeyEvent += SubWinThirdKeyEvent;
         }
 
-        void SetParentAbove()
+        private void SetParentAbove()
         {
-            CreateSubWin3();
-            subWin3.SetParent(mainWin, false);
+            CreateSubWinThird();
+            subWinThird.SetParent(mainWin, false);
         }
 
-        void SetParentBelow()
+        private void SetParentBelow()
         {
-            CreateSubWin3();
-            subWin3.SetParent(mainWin, true);
+            CreateSubWinThird();
+            subWinThird.SetParent(mainWin, true);
         }
 
-        public void subWin3_KeyEvent(object sender, Window.KeyEventArgs e)
+        private void SubWinThirdKeyEvent(object sender, Window.KeyEventArgs e)
         {
             if (e.Key.State == Key.StateType.Down)
             {
-                log.Fatal(tag, $"key down! key={e.Key.KeyPressedName}");
+                log.Debug(tag, $"key down! key={e.Key.KeyPressedName}");
 
                 switch (e.Key.KeyPressedName)
                 {
-                    case "6":
+                    case key6:
```

Comment:
if key 6, 7 means parentabove, below,
what about name them, KeyParentAbove, KeyParentBelow?
also,
Key is constant variable,
so I personally prefer to name constant with all upper case like,
KEY_PARENT_ABOVE
KEY_PARENT_BELOW
KEY_NUM_1
KEY_ESCAPE
..

Reply:
Thank for review. I fixed it.